### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves N/A

**Overall change:**
 @bbc/psammead-byline                 ^1.0.4  →   ^1.0.5
 @bbc/psammead-rich-text-transforms   ^2.0.1  →   ^2.0.2
 @testing-library/react              ^10.0.4  →  ^10.2.1
 @testing-library/react-hooks         ^3.2.1  →   ^3.3.0
 eslint-plugin-import                ^2.20.2  →  ^2.21.1
 lint-staged                         ^10.2.7  →  ^10.2.9
 react-helmet                         ^6.0.0  →   ^6.1.0
 stylelint                           ^13.5.0  →  ^13.6.0
 eslint    ^6.8.0  →  ^7.2.0
 gh-pages   2.2.0  →   3.0.0

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~~Automated jest tests added (for new features) or updated (for existing features)~~
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,37 +111,37 @@
...
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-es5": "^1.5.0",
-    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-import": "^2.21.1",
     "eslint-plugin-json": "^2.1.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-react": "^7.20.0",
-    "gh-pages": "2.2.0",
+    "gh-pages": "3.0.0",
```
Comment (chris-hinds) [inline package.json:123]: Do we know if there are any breaking changes in this major version bump?

Comment (j-pendlebury) [reply package.json:123]: no breaking changes, only says that tests will no longer run on Node < 10 ([full update here](https://github.com/tschaub/gh-pages/releases/tag/v3.0.0))

Comment (chris-hinds) [reply package.json:123]: brill, thanks for checking :)

**Review by chris-hinds — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Comment by jamesdonoh** [comment]
I'm curious, what is the psammead-locales eslint issue?

**Comment by j-pendlebury** [comment]
> I'm curious, what is the psammead-locales eslint issue?

eslint didn't like the formatting of the file. Nothing big but it showed up in Jenkins and caused it to fail 👎

**Pull request merged**